enatega-multivendor-app/src/screens/Menu/Menu.js

```diff
@@ -270,22 +261,42 @@ function Menu({ route, props }) {
     else {
       return (
         <View style={styles().emptyViewContainer}>
-          <TextDefault textColor={currentTheme.fontMainColor}>
-            {t('noRestaurants')}
-          </TextDefault>
+          <View style={styles().emptyViewBox}>
+            <TextDefault bold H4 center textColor={currentTheme.fontMainColor}>
+              We are not currently available in your area
+            </TextDefault>
+            <TextDefault textColor={currentTheme.fontMainColor} center>
+              No restaurant currently offers delivery in your area
+            </TextDefault>
+          </View>
         </View>
       )
     }
   }
+  const errorView = () => {
```

Comment:
I think its generic component and will remain same for each request so make it as a separate component.